packages/expo-updates/e2e/__tests__/setup/project.js

```diff
@@ -0,0 +1,143 @@
+const spawnAsync = require('@expo/spawn-async');
+const fs = require('fs/promises');
+const path = require('path');
+
+async function setupAsync(workingDir, repoRoot, runtimeVersion) {
+  // initialize project
+  await spawnAsync('expo-cli', ['init', 'updates-e2e', '--yes'], {
+    cwd: workingDir,
+    stdio: 'inherit',
+  });
+  const projectRoot = path.join(workingDir, 'updates-e2e');
+
+  // add local dependencies
+  let packageJson = JSON.parse(await fs.readFile(path.join(projectRoot, 'package.json'), 'utf-8'));
+  packageJson = {
+    ...packageJson,
+    resolutions: {
+      ...(packageJson.resolutions ?? {}),
```

Comment:
This works fine, OK to spread undefined in an object (but not in an array)
```suggestion
      ...packageJson.resolutions,
```